marketplace/cloudinary-assets/src/components/cloudinaryField/cloudinaryField.tsx

```diff
@@ -102,7 +102,7 @@ export class CloudinaryField extends React.Component<Props, State> {
             size="small"
             onClick={this.onCloudinaryDialogOpen}
             disabled={isDisabled}>
-            {btnTxt}
+            Select or Upload file on Cloudinary
```

Comment:
Not sure about the context, but I think this would be better.
```suggestion
            Select or upload a file on Cloudinary
```